wpilibc/shared/src/Commands/WaitCommand.cpp

```diff
@@ -9,8 +9,18 @@
 
 using namespace frc;
 
+/**
+ * Creates a new WaitCommand with the given name and timeout.
+ *
+ * @param name    the name of the command
+ * @param timeout the time (in seconds) before this command "times out"
+ */
 WaitCommand::WaitCommand(double timeout)
     : TimedCommand("Wait(" + std::to_string(timeout) + ")", timeout) {}
-
+/**
```

Comment:
Something like this is the domain of clang-format. It probably doesn't ensure newlines around functions. http://stackoverflow.com/questions/27121457/line-breaks-between-function-definitions